R/type-github.R

```diff
@@ -11,6 +11,10 @@
 parse_remote.remote_specs_github <- function(specs, config, ...) {
 
   pds <- re_match(specs, github_rx())
+  if (is.na(pds$.match)) {
```

Comment:
Not sure if there is a better way to do this, basically if the regular regex fails we try the url one.

Reply:
I think it is good.